### PR TITLE
Generate Runner/Fn API protos needed for Fn harness

### PR DESCRIPTION
**Pull request by robertwb**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

**Comment by robertwb** [comment]
R: @aaltay 

I spent some time trying to integrate this into maven, but was never able to get it to a good state (and in particular there's no way to generate the grpc bindings which are needed as well). I'll puruse this more, but for now want to unblock getting the Fn API work into the repo.

File: sdks/python/apache_beam/pipeline.py [hunk 1]
```diff
@@ -345,8 +345,9 @@ def from_runner_api(proto, runner, options):
     p = Pipeline(runner=runner, options=options)
     from apache_beam.runners import pipeline_context
     context = pipeline_context.PipelineContext(proto.components)
+    root_transform_id, = proto.root_transform_ids
```
Comment (aaltay) [inline sdks/python/apache_beam/pipeline.py]: curious, why not `proto.root_transform_ids[0]`?

Comment (robertwb) [inline sdks/python/apache_beam/pipeline.py]: To ensure there's only one. (If I indexed, I might silently drop any others. Whether we keep this as a list or go back to a singleton is TBD.)

File: sdks/python/run_pylint.sh [hunk 2]
```diff
@@ -46,7 +46,7 @@ EXCLUDED_GENERATED_FILES=(
 "apache_beam/io/gcp/internal/clients/storage/storage_v1_client.py"
 "apache_beam/io/gcp/internal/clients/storage/storage_v1_messages.py"
 "apache_beam/coders/proto2_coder_test_messages_pb2.py"
-"apache_beam/runners/api/beam_runner_api_pb2.py")
+"apache_beam/runners/api/*.py")
```
Comment (robertwb) [inline sdks/python/run_pylint.sh]: I'll expand this out if needed.

**Comment by robertwb** [comment]
retest this please

**Comment by robertwb** [comment]
Jenkins: retest this please

**Comment by robertwb** [comment]
Jenkins is still down, and the travis Java tests are perma-red, but the Python ones are passing. Merging.

**Comment by aaltay** [comment]
@jasonkuster @davorbonaci Could we disable the Java tests in Travis? The tests are permared, but Python tests are still providing a useful signal.

**Comment by davorbonaci** [comment]
(yes. soon!)

**Pull request closed without merging**